pkg/sensors/tracing/generickprobe.go

```diff
@@ -1527,6 +1527,16 @@ func handleMsgGenericKprobe(m *api.MsgGenericKprobe, gk *genericKprobe, r *bytes
 			arg.Value = output.Value
 			arg.Label = a.label
 			unix.Args = append(unix.Args, arg)
+		case gt.GenericLinuxBinprmType:
+			var arg api.MsgGenericKprobeArgLinuxBinprm
+
+			arg.Index = uint64(a.index)
```

Comment:
would cast work vs constructing a temporary object on the stack?

Reply:
Hmmm, not sure what you mean here?

Reply:
you can ignore this comment.